views/Products/Create/AddImageSVG.tsx

```diff
@@ -0,0 +1,16 @@
+export const ImageSVG = (
```

Comment:
Please change this name to AddImageSVG

Reply:
Ok, I didn't add that but let me fix it.

Reply:
Done